### PR TITLE
Shielded assets table UI fixes

### LINKED ISSUE #933: [BUG] Show available balance in correct format
### Overview
Show available balance in `Shielded asset` and `Available spend notes` table in the correct format.

Screenshot 2023-01-29 at 2 08 04 AM
Screenshot 2023-01-29 at 2 08 10 AM

### LINKED ISSUE #943: [UI-COMPONENTS] Changes required in shielded assets table
- [ ] Fix text case in the footer.

Expected:
Screenshot 2023-01-30 at 4 10 26 AM

Current:
Screenshot 2023-01-30 at 4 10 40 AM

- [ ] Replace `Available Balance` with `Total Balance`.

Expected:
Screenshot 2023-01-30 at 4 13 14 AM

Current:
Screenshot 2023-01-30 at 4 13 07 AM

- [ ] Composition column should show both asset's icons and should be bigger.

Expected:
Screenshot 2023-01-30 at 4 14 21 AM

Current:
Screenshot 2023-01-30 at 4 14 30 AM

- [ ] Show balance with same decimal places?

Current:
Screenshot 2023-01-30 at 4 17 54 AM

### PR DESCRIPTION
**Pull request by devpavan04**
---
name: Pull request
about: Proposed changes
title: Shielded assets table UI fixes
labels: needs review 👓
assignees: @devpavan04 

---

## Summary of changes
- Updates table pagination text
- Replaces available balance with total balance in table column label
- Composition icons in table are now bigger
- Balance is table is now fixed to six decimal places

### Proposed area of change
- [x] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
- Closes #943 
- Closes #933 

### Screen Recording
https://user-images.githubusercontent.com/53374218/215847960-6dc10d88-4d96-4bcc-b860-bae1d99e8dbb.mov

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/bridge-dapp/src/containers/ShieldedAssetsTableContainer/ShieldedAssetsTableContainer.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ const staticColumns: ColumnDef<ShieldedAssetDataType, any>[] = [
   }),
 
   columnHelper.accessor('availableBalance', {
-    header: 'Available Balance',
+    header: 'Total Balance',
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/containers/ShieldedAssetsTableContainer/ShieldedAssetsTableContainer.tsx:106]:
```suggestion
    header: 'Balance',
```

File: apps/bridge-dapp/src/containers/ShieldedAssetsTableContainer/ShieldedAssetsTableContainer.tsx [hunk 2]
```diff
@@ -275,7 +275,7 @@ export const ShieldedAssetsTableContainer: FC<
         tableProps={table as RTTable<unknown>}
         isPaginated
         totalRecords={data.length}
-        title="Shielded Asset"
+        title="available assets"
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/containers/ShieldedAssetsTableContainer/ShieldedAssetsTableContainer.tsx:278]:
```suggestion
        title="assets"
```

File: apps/bridge-dapp/src/containers/SpendNotesTableContainer/SpendNotesTableContainer.tsx [hunk 3]
```diff
@@ -195,7 +198,7 @@ export const SpendNotesTableContainer: FC<SpendNotesTableContainerProps> = ({
         tableProps={table as RTTable<unknown>}
         isPaginated
         totalRecords={data.length}
-        title="Spend Notes"
+        title="available notes"
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/containers/SpendNotesTableContainer/SpendNotesTableContainer.tsx:201]:
```suggestion
        title="notes"
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Lets just use 'Balance' for table header for both tabs. Can we also centre the items in the table?

**Review by AtelyPham — COMMENTED** [review]
LGTM 👍 Please fix the ci failing here @devpavan04

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**